src_plugins/enum/ppx_deriving_enum.cppo.ml

```diff
@@ -5,6 +5,10 @@ open Parsetree
 open Ast_helper
 open Ppx_deriving.Ast_convenience
 
+#if OCAML_VERSION < (4, 08, 0)
+module Stdlib = Pervasives
+#endif
```

Comment:
If we want to get rid of cppo, we could instead add the following binding at the beginning of the module:
```ocaml
let stdlib_compare = compare
```

Reply:
Agree on moving to stdlib-shims. It will solve this particular `#if`, but there are some others, unrelated to Stdlib.